src/main/java/net/dv8tion/jda/api/entities/MessageChannel.java

```diff
@@ -1565,7 +1565,68 @@ default MessagePaginationAction getIterableHistory()
         Checks.notNull(message, "Message");
         return getHistoryBefore(message.getId(), limit);
     }
-
+    
+    /**
+     * Retrieves messages from the beginning of this MessageChannel.
+     * The {@code limit} determines the amount of messages being retrieved.
+     * 
+     * <p><b>Example:</b>
```

Comment:
You can use a header tag here like `<h2>Example</h2>`

Reply:
Most methods also have it like that so I stuck with that.
I will replace all of those occurrences with a proper header tag if that's okay for you.

Reply:
It doesn't need to be changed everywhere, the resulting javadoc is the same. However, only old docs should have this style while the new ones use headers instead.